test/snippet/argument_parser/validators_input_directory.cpp

```diff
@@ -0,0 +1,32 @@
+//! [usage]
+#include <seqan3/argument_parser/all.hpp>
+#include <seqan3/io/stream/debug_stream.hpp>
+#include <seqan3/std/filesystem>
+
+int main(int argc, const char ** argv)
+{
+    seqan3::argument_parser myparser("Test", argc, argv); // initialize
+
+    //! [validator_call]
+    std::filesystem::path myfile;
```

Comment:
```suggestion
    std::filesystem::path mydir{};
```
always initialize values for the parser! otherwise the default values printed on the help page are weird.